demo/DocsDecorator/DocsDecorator.scss

```diff
@@ -0,0 +1,15 @@
+$root: '.docs-decorator';
+
+#{$root}#{$root}#{$root}#{$root}#{$root} {
```

Comment:
what caused it?)

Reply:
honestly I didn't check it, just copied from uikit)
https://github.com/gravity-ui/uikit/blob/4519ea5b3baa3a0e665a8b477e73ef8ea5cb44a3/src/demo/DocsDecorator/DocsDecorator.scss#L10